docs/data/toolpad/reference/components/metric.md

```diff
@@ -0,0 +1,18 @@
+<!-- This file has been auto-generated using `yarn docs:build:api`. -->
+
+# Metric
+
+<p class="description">API docs for the Toolpad Metric component.</p>
+
+The Metric component lets you display values.
```

Comment:
Should we elaborate a bit more on possible typical use cases in this description?